docker/etna-base/Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV DOCKERIZE_VERSION 0.5.0
 ENV NODE_VERSION 12.18.1
 ENV DOCKERIZE_URL https://github.com/jwilder/dockerize/releases/download/v${DOCKERIZE_VERSION}/dockerize-linux-amd64-v${DOCKERIZE_VERSION}.tar.gz
 ENV RACK_ENV development
+ENV PATH "/app/bin:/app/node_modules/.bin:/app/vendor/bundle/$RUBY_VERSION/bin:$PATH"
```

Comment:
Pushing the path into the base image rather than the entry points was better for the resulting image.